massa-ledger/src/config.rs

```diff
@@ -9,8 +9,4 @@ use std::path::PathBuf;
 pub struct LedgerConfig {
     /// initial SCE ledger file
     pub initial_sce_ledger_path: PathBuf,
```

Comment:
`initial_sce_ledger_path` is never read. Since `FinalStateConfig` now contains most of the information is `LedgerConfig` still needed?

Reply:
My bad it's used [here](https://github.com/massalabs/massa/blob/massa-execution-state/massa-ledger/src/ledger.rs#L80)